binaryaudit/dnf.py

```diff
@@ -81,11 +81,27 @@ def download(key, source_dir, name, old_rpm_dict):
     if old_rpm_name == "":
         return old_rpm_name
     old_rpm_name = old_rpm_name.replace("\r\n", "")
-    docker_loc, docker_loc_exit_code = run.run_command_docker(["/usr/bin/dnf", "repoquery", "--quiet",
-                                                              "--location", "--latest-limit=1", name], None, subprocess.PIPE)
-    url = docker_loc.stdout.read().decode('utf-8')
+    i = 0
+    for i in range(3):
+        docker_loc, docker_loc_exit_code = run.run_command_docker(["/usr/bin/dnf", "repoquery", "--quiet", "--location",
+                                                                  "--latest-limit=1", name], None, subprocess.PIPE)
+        url = docker_loc.stdout.read().decode('utf-8')
+        if url == "":
+            i += 1
```

Comment:
Not sure this is needed, `i` is visible and keeps the value after the loop. Just having a break at success should be enough.

Thanks